packages/docs/blog/2025-12-30-roadmap-for-2026.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Roadmap for 2026
+description: We were able to accomplish a lot in 2025. Lets look back over the year and also look at whats to come.
+date: 2025-12-30T10:00
+slug: roadmap-for-2026
+tags: [announcement]
+hide_table_of_contents: false
+authors: youngcw
+---
+
+We were able to accomplish a lot in 2025.
+Lets look back over the year and also look at whats to come.
+
+<!--truncate-->
+
+## 2025 in review
+Its always fun to look back and see all the progress made in Actual over the course of a year.
+Its easy to forget how many of the features we love weren't available back at the beginning of the year.
+
+My favorites in approximately chronological order are:
+- OIDC and Multi-user support
+- Adjustable upcoming time frame for schedules
+- Multiple new report types
+- NPM package for the sync server
+- Translations
+- Bank sync management page
+- Desktop app improvements such as the embedded server
+- Auto-hold income
+- Merge transactions
+- Transaction tags
+- Mobile pages for Rules, Payees, Schedules, and Bank Sync
+- Budget Currencies (still experimental)
+- Quickly add single schedules from the account view
+
+**Note on Translations:**  Translation in a never ending project.
+Every month there are new or modified strings that need translated.
+We could use as many contributors to translation as we can get.
+If you are interested in helping translate Actual into other languages see [our translation guide].
+
+If you remember our roadmap from last year, you will see that we were able to accomplish all but one of our goals, plugins.
+I will touch on that more below.
+
+## 2026 Wish list
+
+**Plugins**: We have been working on adding plugins to allow users to more easily add functionality to the app without the need to add said feature to the base app.
+The feature is not quite ready as plugins have a tendency to break existing features, which slows down development a great deal.
+We are doing everything we can to not break existing features that we know you rely on.
+Our hope is to have the feature ready for use in the first few months of 2026.
+
+Here is a sampling of some of the plugins we hope will be developed
+- Many options for bank sync providers. We will migrate the existing providers to plugins and there are other providers waiting for plugins to be available for proper integration into the app.
+- Custom accounts types
+- Themes
+- New report types
+- Other specialty functionality
```

Comment:
I don't think we'll be able to do most of these things with the first release of plugins. Maybe drop this list so people don't get too excited & then disappointed with the release?